### PR TITLE
API-5954-test-support-lib

### PR DESCRIPTION
**Pull request by jhulbert-va**
# [API-5954](https://vajira.max.gov/browse/API-5954)

### CODE REVIEW

File: test-support/pom.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>gov.va.api.health</groupId>
+    <artifactId>health-apis-parent</artifactId>
+    <version>7.0.23</version>
+    <relativePath/>
+  </parent>
+  <groupId>gov.va.api.health</groupId>
+  <artifactId>test-support</artifactId>
+  <version>6.0.7-SNAPSHOT</version>
```
Comment (jblefkove-va) [inline test-support/pom.xml:12]: will need to go to next SNAPSHOT

File: test-support/spotbugs-excludes.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<FindBugsFilter>
+    <!-- Exclusions can be defined here and should include a comment on why the finding can be ignored -->
+    <!--
+  Ignore Lombok-generated code.
+-->
```
Comment (jblefkove-va) [inline test-support/spotbugs-excludes.xml:6]: comments are misaligned

File: test-support/src/main/java/gov/va/api/health/fhir/testsupport/ResourceVerifier.java [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+  private final Class<?> bundleClass;
+
+  private final TestClient testClient;
+
+  private final Class<?> operationOutcomeClass;
+
+  /**
+   * If the response is a bundle, then the query is a search. We want to verify paging parameters
+   * restrict page >= 1, _count >=1, and _count <= 20
```
Comment (jblefkove-va) [inline test-support/src/main/java/gov/va/api/health/fhir/testsupport/ResourceVerifier.java:40]: isn't it count <= 100 now?

Comment (jhulbert-va) [reply test-support/src/main/java/gov/va/api/health/fhir/testsupport/ResourceVerifier.java:40]: With this being in a common area, should we make that configurable?

Comment (bschofield-va) [reply test-support/src/main/java/gov/va/api/health/fhir/testsupport/ResourceVerifier.java]: yes

**Review by bschofield-va — APPROVED** [review]

**Pull request merged**